Backend/dev/mock-servers/services/gridline.py

```
@@ -14,6 +14,53 @@
 WEBHOOK_SECRET = "xxxxxxx"
 CALLBACK_DELAY = 0.5
 
+# Court Record Check (CRC) webhook is merchant/city-scoped and uses the
+# Verification_Idfy secret. Hardcoded for the local NY Bangalore test env
+# (the async submit body carries only group_id=driverId, not merchant/city).
+# Routed via caddy (8016), which proxies the webhook path to the driver-app.
+CRC_WEBHOOK_URL = "http://localhost:8016/NAMMA_YATRI_PARTNER/Bangalore/service/idfy/verification"
+CRC_WEBHOOK_SECRET = "test-secret"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "CRC_WEBHOOK_SECRET"

(S105)